README.md

```diff
@@ -34,6 +34,13 @@ export GEOM=mygeom
 export OPTICKS_EVENT_MODE=DebugLite
 ```
 
+Also the following environment variables need to be set:
+
+```
+export OPTICKS_MAX_PHOTON=M60
+export OPTICKS_MAX_SLOT=M60
+```
+
```

Comment:
Environment variables can/should be specified in config/*.json files https://github.com/BNLNPPS/eic-opticks/tree/main/config